src/arkit.cs

```diff
@@ -190,6 +190,7 @@ public enum ARCoachingGoal : long {
 		HorizontalPlane,
 		VerticalPlane,
 		AnyPlane,
```

Comment:
missing `[iOS (15,0)]`

Reply:
I noticed just when I did it :/ I'm rusty